Component/Diffusion/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 3.12)
+project(Sofa.Component.Diffusion LANGUAGES CXX)
+
+set(SOFACOMPONENTDIFFUSION_SOURCE_DIR "src/sofa/component/diffusion")
+
+set(HEADER_FILES
+	${SOFACOMPONENTDIFFUSION_SOURCE_DIR}/config.h.in
```

Comment:
indentation

Reply:
What indentation ❓

Reply:
The way I see it in github, line 7 is not indented like the following lines

Reply:
weird i dont see it in the diff here, but I view the file directly I see it.
thanks !